addons/stock/report/report_stockpicking_operations.xml

```diff
@@ -177,7 +177,7 @@
                                                     <span t-out="format_number(line.quantity)">3.00</span>
                                                     <span t-field="line.product_uom_id" groups="uom.group_uom">units</span>
                                                     <span t-if="line.move_id.packaging_uom_id">
-                                                        (<span t-field="line.product_packaging_qty" t-options='{"widget": "integer"}'/> <span t-field="line.move_id.packaging_uom_id.name"/>)
+                                                        (<span t-field="line.move_id.packaging_uom_qty" t-options='{"widget": "integer"}'/> <span t-field="line.move_id.packaging_uom_id.name"/>)
```

Comment:
added `move_id` as it looks fairly logical to use it, but not sure I should :thinking: